src/odemis/gui/cont/cryo_project.py

```
@@ -0,0 +1,148 @@
+def save_project(project_dir: str, features, overviews) -> None:
+    """
+    Save the whole features list directly to the file
+    :param project_dir: directory to save the file to (typically project directory)
+    :param features: all the features to serialize
+    :param overviews: TODO
+    """
+    filename = os.path.join(project_dir, PROJECT_NAME)
+    with open(filename, "w") as jsonfile:
+        json.dump(get_project_dict(features, overviews), jsonfile)
+
+def read_project_file(project_file) -> List["CryoFeature"]:
```

Ruff (0.15.10)

[error] Undefined name `CryoFeature`

(F821)